js-exercises/sleep-promise/sleepPromise.js

```diff
@@ -0,0 +1,8 @@
+const sleep = async ms => {
+  await new Promise((resolve) => {
+    setTimeout(resolve, ms);
+  });
+};
```

Comment:
this function will not let other code to run if I have written

`sleep(2000).then(function() {
  console.log("2 seconds later …");
});

somethingElse()`